sqlmypy.py

```diff
@@ -141,14 +142,23 @@ def decl_info_hook(ctx):
 
         Base = declarative_base()
     """
+    cls_instances = []  # type: List[Instance]
+
+    if 'cls' in ctx.call.arg_names:
+        declarative_base_cls_arg = ctx.call.args[ctx.call.arg_names.index("cls")]
+        if isinstance(declarative_base_cls_arg, TupleExpr):
+            cls_instances = [Instance(item.node, []) for item in declarative_base_cls_arg.items]
```

Comment:
You need to guard both here and below  with `isinstance(item, RefExpr) and isinstance(item.node, TypeInfo)`.

(This reminds me that we need to type-check the plugin with mypy in CI, currently it is not, mypy would caught such mistakes.)

Reply:
what do you mean by guard ? an assert ?

Reply:
I mean `if isinstance()`.